<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Sans L</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QPushButton" name="Login_button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>177</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="Status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>205</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>86</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>USERNAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLineEdit" name="Username_field"/>
      <widget class="QLineEdit" name="Password_field">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="database_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
